simconvoi.h

```diff
@@ -1017,6 +1026,9 @@ class convoi_t : public sync_steppable, public overtaker_t
 
 	bool is_coupled() const { return state==COUPLED  ||  state==COUPLED_LOADING; }
 	bool is_waiting_for_coupling() const;
+	void set_will_coupling_convoi(convoihandle_t convoi_coupling_undergo);
```

Comment:
(nit) ここはただのsetterなので、特に引数名を設けなくても十分意味はわかるかなと思います。
```suggestion
	void set_will_coupling_convoi(convoihandle_t);
```